### DISCUSSION TITLE
Welcome to AsyncMQTT_Generic Discussions!

### DISCUSSION THREAD

**Discussion by khoih-prog** [khoih-prog/AsyncMQTT_Generic#3] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gendiaaa** [comment]
hello Sir ,
 i want to know how to setup my User  and password for mqtt client in your Example for Esp32 please
